assignments/prototypes.js

```diff
@@ -23,6 +37,37 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(object) {
+  GameObject.call(this, object);
+  this.healthPoints = (object.healthPoints * object.vitality) / 2;
+  this.strength = object.strength;
+  this.vitality = object.vitality;
+  this.dead = false;
+}
+
+CharacterStats.prototype = GameObject.prototype;
```

Comment:
dont forget that Object.create function, writing like this is very prone to bugs, and you could run into a lot of problems with not knowing what the object is set to, or having the address on another object.
```suggestion
CharacterStats.prototype = Object.create(GameObject.prototype);
```
this sets it equal to a new object as opposed to an existing one
you should be doing this throughout the entire project.